sbor-derive-common/src/utils.rs

```diff
@@ -41,13 +41,13 @@ pub fn print_generated_code<S: ToString>(kind: &str, code: S) {
 pub fn extract_attributes(
     attrs: &[Attribute],
     name: &str,
-) -> Option<HashMap<String, Option<String>>> {
+) -> Option<BTreeMap<String, Option<String>>> {
```

Comment:
Why not `NonIterMap`?

Reply:
This is a macro impl, and I figured that performance does not matter at all. While pulling in a dependency to `NonIterMap` could be problematic, says David (see https://rdxworks.slack.com/archives/C01M90Y2448/p1680038448642949?thread_ts=1680028951.916639&cid=C01M90Y2448).